app/views/content/steps-to-become-a-teacher.md

```diff
@@ -88,7 +88,13 @@
     international-content:
       text: There are more steps to consider if you're <a href="/non-uk-teachers">a non-UK citizen</a>.
       color: grey
+    still-time-to-apply:
+      title: Still time to apply
```

Comment:
```suggestion
      title: Start your training this September
```
am just wondering if the current heading is a bit unclear? could something like this work a bit better? or if not, do we at least need to add '**There's** still time to apply **this year**' or something??